lua/modules/menu.lua

```diff
@@ -42,6 +49,14 @@ modalWasShown = false
 alertWasShown = false
 cppMenuIsActive = false
 chatDisplayed = false -- when false, only a mini chat console is displayed in the top bar
+_DEBUG = false
+_DebugColor = function()
+	return Color(
+		150 + math.floor(math.random() * 105),
```

Comment:
here we can replace by a simpler form `math.random(150,255)`

Reply:
Done!